tests/Rule/EnsureExplicitNullableTypesTest.php

```diff
@@ -44,14 +44,21 @@ public static function validProvider(): iterable
         $validCases = [
             'function foo(int $bar = 23)',
             'function foo(?int $bar = null)',
+            'function foo(?\\Throwable $exception = null)',
+            'function foo(\\Throwable|null $exception = null)',
+            'function foo(?Foo\\Bar $bar = null)',
+            'function foo(mixed $bar = null)',
             'function foo(int|null $bar = null)',
             'function foo(int|string|null $bar = null)',
         ];
 
         foreach ($validCases as $validCase) {
             yield $validCase => [
                 NullViolation::create(),
-                new RstSample(['.. code-block:: php', $validCase]),
+                new RstSample([
+                    '.. code-block:: php',
+                    '    '.$validCase,
```

Comment:
just for clarification, this indention is needed, otherwise the code does not _belong_ to the code block

Reply:
Oh ok ! Thanks for the explanation :)